numpy/_core/numerictypes.py

```diff
@@ -359,6 +360,90 @@ def issubsctype(arg1, arg2):
     return issubclass(obj2sctype(arg1), obj2sctype(arg2))
 
 
+@set_module('numpy')
+def isdtype(dtype, kind):
+    """
+    Returns a boolean indicating whether a provided dtype is of a specified
```

Comment:
Docstrings should start with a single line followed by a blank line. That way the autosummary tables don't overflow.

Reply:
Done!